src/cloudant/_common_util.py

```diff
@@ -177,7 +177,7 @@ def python_to_couch(options):
     translation = dict()
```

Comment:
It doesn't allow me to comment on line 174 but we should add something like:
```
:param encoder: Custom encoder, defaults to None
```

Reply:
Thanks! I added in https://github.com/cloudant/python-cloudant/pull/501/commits/88a1703c9609c4f1d3e7b3fa1f23040088e41417